lib/iiif_manifest/v3/manifest_builder/content_builder.rb

```diff
@@ -11,6 +11,7 @@ def initialize(display_content, iiif_annotation_factory:, body_builder_factory:)
         end
 
         def apply(canvas)
+          annotation['id'] = canvas['id'].to_s.gsub(/manifest.*/, "annotation/#{annotation.index}")
```

Comment:
Why did you decide to make the annotation id a first level entity (e.g. `http://test.host/books/book-77/annotation/27694452-fb7a-4eb2-b006-f62b54e1c44a`) instead of a child of the annotation page (e.g. `http://test.host/books/book-77/manifest/canvas/test-22/annotation_page/1a6d9e1d-7b3d-48cd-8439-061d7231b62b/annotation/27694452-fb7a-4eb2-b006-f62b54e1c44a`)?

Reply:
In the example given by the Presentation 3.0 documentation they have the annotation id as a first level entity, rather than a child of the annotation_page: https://iiif.io/api/presentation/3.0/#56-annotation. However, it is not specified in the documentation itself that having it as a first level entity is necessary, so if having it as the child of annotation_page makes more sense we can do that.

Reply:
On further thought, especially since they are unresolvable anyway, it would probably be better to go for consistency with the rest of the manifest/matching the nested JSON. I will go ahead and push an update with the nested URI in a little bit.